template.tpl

```diff
@@ -2530,6 +2523,13 @@ ___WEB_PERMISSIONS___
         "versionId": "1"
       },
       "param": [
+        {
+          "key": "allowedKeys",
+          "value": {
+            "type": 1,
+            "string": "specific"
+          }
+        },
```

Comment:
This seems to just be an update to the export format for this permission, not any actual change in the permissions as part of this change.